src/dbt/kipptaf/models/people/staging/stg_people__manager_pm_score_averages.sql

```diff
@@ -0,0 +1,167 @@
+with
+    measures_long as (
+        select
+            observer_employee_number,
+            academic_year,
+            form_term,
+            row_score_value,
+            -- trunk-ignore-begin(sqlfluff)
+            case
+                when
+                    measurement_name
+                    = 'This teammate addresses challenges directly and productively.'
+                then 'SO1'
+                when
+                    measurement_name
+                    = 'This teammate chooses to make work a positive and joyful experience for students, self, and others.'
+                then 'SO2'
+                when
+                    measurement_name
+                    = 'This teammate demonstrates professional responsibilities and individual commitments.'
+                then 'SO3'
+                when
+                    measurement_name
+                    = 'This teammate makes others feel included, loved, and valued, even across cultural differences.'
+                then 'SO4'
+                when
+                    measurement_name
+                    = 'This teammate meets professional expectations for presence.'
+                then 'SO5'
+                when
+                    measurement_name
+                    = 'This teammate meets professional expectations for punctuality.'
+                then 'SO6'
+                when
+                    measurement_name
+                    = 'This teammate seeks feedback and takes it seriously.'
+                then 'SO7'
+                when
+                    measurement_name
+                    = 'This teammate supports the collective work of the team and meets team commitments.'
+                then 'SO8'
+                else concat('ETR', left(measurement_name, 2))
+            end as measurement_code,
+        -- trunk-ignore-end(sqlfluff)
+        from {{ ref("rpt_tableau__schoolmint_grow_observation_details") }}
```

Comment:
use the schoolmint staging tables or refactor the data you need from the tableau view to a intermediate view that both reporting views can use

Reply:
done